src/org/zaproxy/zap/extension/ascanrules/TestRemoteFileInclude.java

```diff
@@ -247,7 +247,10 @@ public void scan(HttpMessage msg, String param, String value) {
             }
 
         } catch (Exception e) {
-            log.error("Error scanning parameters for Path Traversal: " + e.getMessage() + " Caused by: " + e.getCause());
+        	if (log.isDebugEnabled()) {

```

Comment:
Due to the previous logging, I don't know what the actual exception is (and I don't have a test case for it), it had been logging as a failure caused by null....